User-Help/accessibility.md

```diff
@@ -0,0 +1,37 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| accessibility | About the Network |
+
+# Codidact Accessibility Statement
+
+Sharing information and being part of a community are the two fundamental goals of the Codidact Project. Everyone deserves access to information and to be a part of a community, and with that in mind,  the Codidact Foundation is committed to accessibility in all areas of the Codidact Project.
+
+## Who is responsible for accessibility?
+
+Accessibility is the responsibility of everyone, from the people writing posts to the people working on the website. However, the chief person responsible for accessibility in the Codidact Foundation is Mithical, Co-Lead for User Documentation and a member of the Board of Directors. Questions about Codidact's accessibility can be directed to support@codidact.org.
+
+## I'm encountering an accessibility problem, what do I do?
+
+Please report any accessibility problems you have on [Codidact Meta](https://meta.codidact.com), using the [bug] and [accessibility] tags. That makes sure that we're aware of the problem, and we'll do our best to fix it as soon as reasonably possible with our available resources.
+
+## Technical details: Codidact Accessibility Policy
+
+This Accessibility Policy outlines the technical details of our commitment to accessibility and sets expectations for what is in scope of that commitment.
+
+### The Codidact Network
+
+The platform that the Codidact Network runs on, known as QPixel, should meet both the Web Content Accessibility Guidelines (WCAG) 2.2 and the Authoring Tool Accessibility Guidelines (ATAG) 2.0 standards at the AA conformance level.
+
+Since the platform is open-source and volunteer operated, setting strict timetables isn't realistic. However, new changes to the platform should be made with these standards in mind, including using Accessible Rich Internet Applications (WAI-ARIA) features.
```

Comment:
Nit, only because we need to edit something else anyway: volunteer-operated, with a hyphen (aligns with open-source)